platforms/aws-graviton/sync-from-remote.sh

```diff
@@ -9,11 +9,8 @@ echo "Sync: Syncing code from EC2"
 echo "Sync: Removing existing local code"
 rm -rf ./code
 
-echo "Sync: Removing remote node_modules on server"
-ssh -tt -i ./server-key.pem ec2-user@ec2-54-209-135-27.compute-1.amazonaws.com 'rm -rf /home/ec2-user/aws-graviton/node_modules'
-
 echo "Sync: Copying remote code from EC2 to local"
-scp -i ./server-key.pem -rp ec2-user@ec2-54-209-135-27.compute-1.amazonaws.com:/home/ec2-user/aws-graviton/ .
+rysnc -Pav -e "ssh -i ./server-key.pem" --exclude "node_modules" ec2-user@ec2-54-209-135-27.compute-1.amazonaws.com:/home/ec2-user/aws-graviton/ .
```

Comment:
I had to familiarise myself with this diff, here's what I got:

- Before, we securely transferred the current folder to EC2 via `scp`, specifying the private key file with `-i`, recursively copying folders (with `-r`) while preserving metadata such as access modes and time.
- Now, we use `rsync` - which is kind of an optimised `scp`, so perhaps it will lead to lower transferred traffic. `-P` keeps partially transferred files and shows progress interactively, `-a` is the same as `scp -rp`, and `-v` indicates "verbose mode". We're also skipping the transfer of the `node_modules` folder (with `--exclude`)

Sounds like a win to me, perhaps let's add a couple more comments to these scripts next time :)

Reply:
`--exclude` option was actually main motivator for moving to `rsync`

Reply:
Replaced short `rsync` options with long ones, hope that helps.